src/linear_algebra/general_linear_group.lean

```diff
@@ -186,11 +188,25 @@ lemma to_GL_pos_injective :
 (show function.injective ((coe : GL_pos n R → matrix n n R) ∘ to_GL_pos),
  from subtype.coe_injective).of_comp
 
+/-- Coercing a `special_linear_group` via `GL_pos` and `GL` is the same as coercing striaght to a
+matrix -/
+@[simp]
+lemma coe_GL_pos_coe_GL_coe_matrix (g : special_linear_group n R) :
+  (↑(↑(↑(g : special_linear_group n R) : GL_pos n R) : GL n R) : matrix n n R) = ↑g := rfl
+
+lemma coe_to_GL_pos_det (g : special_linear_group n R) :
+  det ((g : GL_pos n R) : matrix n n R) = (1 : R) :=g.prop
```

Comment:
Can this be `@[simp]`?

Reply:
Lint complains that simp can already prove this. I'm not sure if this means we shouldn't have this lemma or if it shouldn't be a simp lemma.

Reply:
That probably means we shouldn't have this lemma. I pushed a commit that replaces it with one that the linter doesn't complain about

Reply:
I see you added a stronger version of this, thanks!